src/components/CohortBuilder/ParticipantsTableView/ParticipantsTable.js

```diff
@@ -43,27 +46,35 @@ const SelectionCell = ({ value: checked, onCellSelected, row }) => {
 
 const enhance = compose(withState('collapsed', 'setCollapsed', true));
 const CollapsibleMultiLineCell = enhance(({ value: data, collapsed, setCollapsed }) => {
-  // Display a fourth row when there is exactly 4 rows.
+  // Display one row when there is exactly more than one row.
   // Collapsing a single don't save any space.
-  const displayedRowCount = collapsed ? (data.length === 4 ? 4 : 3) : data.length;
-  const displayMoreButton = data.length > 4;
+  const sortedData = union(data);
```

Comment:
We will have to remove this call to union, since it doesn't meet our requirements. Speaking of requirements, we will move the removal of duplicates into another story.

Reply:
@aalex said it's fine now :)